modules/tinymce/CHANGELOG.md

```diff
@@ -66,6 +66,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Disabled nested menu items could still be opened #TINY-7700
 - `imagetools` buttons were incorrectly enabled for remote images without `imagetools_proxy` set #TINY-7772
 - Some table operations would incorrectly cause table row attributes and styles to be lost #TINY-6666
+- The nested menu item chevron icon was not fading when the menu item was disabled #TINY-7700
```

Comment:
Can you please put this at the end? or at the very least not in the middle of related changelog entries.